perf/src/index.html

```diff
@@ -7,7 +7,8 @@
 </head>
 <body>
 <div id="root"></div>
+<div id="control"></div>
```

Comment:
lets introduce comment for the second mount point that would explain what it is used for (and, maybe we could rename it to `control-tools`, `tools-panel` or something like that)